sumatra/decorators.py

```diff
@@ -13,6 +13,7 @@ def main(parameters, [other_args...]):
 """
 from __future__ import unicode_literals
 
+from builtins import str
```

Comment:
``builtins`` is a module from the compatibility layer ``python-future`` that replaces builtin functions. The basic idea behind is that you can write code as if using Python builtins while having Py2/Py3 compatibility. For the case that at one point you drop Py2 compatibility you simply have to remove the import.

In this special case ``str`` is a function that creates python-future's ``new_str`` type in Py2 and an ordinary Py3 `str` in the Py3 case (basically a bypass to the _real_ Py3 `str()` function). ``new_str`` resembles Py3 `str` behaviour in Py2.